golem_messages/message/__init__.py

```diff
@@ -0,0 +1,146 @@
+# For backwards compatibility import all old messages by name
+# pylint: disable=unused-import
+# pylint: disable=cyclic-import
```

Comment:
should we enable these again after this block?

Reply:
done